documentation/docs/system-architecture/design.md

```diff
@@ -216,93 +216,93 @@ Use Case 1 Discription
 sequenceDiagram
     participant Owner as Owner
     participant Discord as Discord Server
-    participant BeRealBot as BeReal Bot
+    participant WhatchaDoinBot as WhatchaDoin Bot
     participant Database as Configuration Database
 
     Owner ->> Discord: Open server "Preferences"
     activate Discord
     Owner ->> Discord: Select "Apps and Integrations"
-    Discord ->> BeRealBot: Install BeReal bot
+    Discord ->> WhatchaDoinBot: Install WhatchaDoin bot
     deactivate Discord
-    activate BeRealBot
-    Owner ->> BeRealBot: Assign roles and privileges
-    Owner ->> BeRealBot: Define moderation and content guidelines
-    Owner ->> BeRealBot: Define prompts culture and vibe
-    Owner ->> BeRealBot: Configure schedule for prompts
-    Owner ->> BeRealBot: Set duration for prompt responses
-    Owner ->> BeRealBot: Set response time limit
-    BeRealBot ->> Database: Save configuration settings
-    deactivate BeRealBot
-    Database -->> BeRealBot: Confirmation
+    activate WhatchaDoinBot
+    Owner ->> WhatchaDoinBot: Assign roles and privileges
+    Owner ->> WhatchaDoinBot: Define moderation and content guidelines
+    Owner ->> WhatchaDoinBot: Define prompts culture and vibe
+    Owner ->> WhatchaDoinBot: Configure schedule for prompts
+    Owner ->> WhatchaDoinBot: Set duration for prompt responses
+    Owner ->> WhatchaDoinBot: Set response time limit
+    WhatchaDoinBot ->> Database: Save configuration settings
+    deactivate WhatchaDoinBot
+    Database -->> WhatchaDoinBot: Confirmation
     activate Database
-    BeRealBot -->> Owner: Configuration settings saved
+    WhatchaDoinBot -->> Owner: Configuration settings saved
     deactivate Database
 
 ```
 <br/><br/>
 
-**Use Case #2**: User Responds to a BeReal bot Prompt
+**Use Case #2**: User Responds to a WhatchaDoin bot Prompt
 <details>
 <summary>
 Use Case 2 Discription
 </summary>
   
-  1. User in the Discord community receives a notification at a random time of day that they have received a prompt from the BeReal bot
+  1. User in the Discord community receives a notification at a random time of day that they have received a prompt from the WhatchaDoin bot
   2. User opens Discord.
   3. User responds to the random prompt by taking a picture and uploading it.
-  4. User replies to the BeReal bot with their response to the prompt, which is sent to the moderator.
-  5. User waits for approval status from the BeReal bot.
+  4. User replies to the WhatchaDoin bot with their response to the prompt, which is sent to the moderator.
+  5. User waits for approval status from the WhatchaDoin bot.
       
 </details>
 
 ```mermaid
 
 sequenceDiagram
     participant User as Discord User
-    participant BeRealBot as BeReal Bot
-    participant Moderator as Moderator
+    participant WhatchaDoinBot as WhatchaDoin Bot
+    actor Moderator as Moderator
```

Comment:
This isn't right, he was just giving an example in the correction doc, a lot of the sequence diagrams are wrong, so I need to fix